src/components/InventoryGroups/Modals/DeleteGroupModal.cy.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import { mount } from '@cypress/react';
+import { Provider } from 'react-redux';
+import { MemoryRouter } from 'react-router-dom';
+import { getStore } from '../../../store';
+import DeleteGroupModal from './DeleteGroupModal';
+
+describe('Delete Group Modal', () => {
+    before(() => {
+        cy.window().then(window => window.insights = {
+            chrome: {
+                isProd: false,
+                auth: {
+                    getUser: () => {
+                        return Promise.resolve({});
+                    }
+                }
+            }
+        });
+    });
+
+    beforeEach(() => {
+        cy.intercept('DELETE', '**/api/inventory/v1/groups/1', {
+            statusCode: 200, body: {
+            }
+        }).as('delete');
+
+        mount(
+            <MemoryRouter>
+                <Provider store={getStore()}>
+                    <DeleteGroupModal isModalOpen={true}/>
+                </Provider>
+            </MemoryRouter>
+        );
+    });
+
+    it('Input is fillable and firing a delete request', () => {
+        cy.get(`div[class="pf-c-check"]`).click();
+        cy.get(`button[type="submit"]`).click();
+        cy.wait('@delete').then((xhr) => {
+            expect(xhr.request.url).to.contain('1');}
+        );
```

Comment:
Since you are intercepting the exact request with the testing ID, then the `then` block can be omitted. The required check will be already done by interceptor. 
```suggestion
        cy.wait('@delete');
```

Reply:
Great! thanks :)